### PR TITLE
[Merged by Bors] - feat: a measurable cylinder is measurable

### PR DESCRIPTION
**Pull request by EtienneC30**
Add this easy lemma.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Constructions/Cylinders.lean [hunk 1]
```diff
@@ -272,6 +272,12 @@ theorem mem_measurableCylinders (t : Set (∀ i, α i)) :
     t ∈ measurableCylinders α ↔ ∃ s S, MeasurableSet S ∧ t = cylinder s S := by
   simp_rw [measurableCylinders, mem_iUnion, exists_prop, mem_singleton_iff]
 
+@[measurability]
+theorem measurableSet_measurableCylinders {s : Set (Π i, α i)} (hs : s ∈ measurableCylinders α) :
```
Comment (j-loreaux) [inline Mathlib/MeasureTheory/Constructions/Cylinders.lean:276]: How about this?
```suggestion
theorem _root_.MeasurableSet.of_mem_measurableCylinders {s : Set (Π i, α i)} (hs : s ∈ measurableCylinders α) :
```

**Review by j-loreaux — COMMENTED** [review]
Please wait for CI to finish before merging.

bors d+

File: Mathlib/MeasureTheory/Constructions/Cylinders.lean (hunk 1 above)
Comment (EtienneC30) [reply Mathlib/MeasureTheory/Constructions/Cylinders.lean]: This is certainly better

**Comment by EtienneC30** [comment]
Thanks!

bors r+

**Pull request closed without merging**